src/static/css/cfpb-chart-builder.less

```diff
@@ -60,18 +60,38 @@
     .chart-label();
   }
 
+  .highcharts-label__tile-map {
+    .chart-label();
+    color: @black;
+    font-family: 'AvenirNextLTW01-Demi',Arial,sans-serif;
+    font-weight: bold;
+    span {
+      position: relative !important;
+      white-space: normal !important;
+    }
+  }
+
   &[data-chart-type="tile_map"] {
     .highcharts-data-label {
 
       color: @black;
       font-family: "AvenirNextLTW01-Demi";
+      font-size: 14px;
+      line-height: 1.4em;
```

Comment:
Line heights do not require a unit type (https://github.com/cfpb/development/blob/master/css.md#property-values)